core/src/test/java/dk/alexandra/fresco/lib/lp/LpBuildingBlockTests.java

```diff
@@ -505,7 +506,7 @@ public void test() {
    * @param value the value
    */
   private static void assertDebugInfoContains(String output, String key, String value) {
-    assertTrue(output.contains(key + ": \n" + value));
+    Assert.assertThat(output, Matchers.containsString(key + ": \n" + value));
```

Comment:
For consistency, please use a static import for `assertThat` here for consistency.